### ISSUE TITLE
Lack of Hover over behavior for starter radio and dropdown

### ISSUE THREAD

**Issue by mbroz2** [OpenLiberty/openliberty.io#2788]
The radio and dropdowns in the starter section of /start is absent.  This could mislead a user into thinking that they are not interactable elements (and perhaps an a11y violation).
![image](https://user-images.githubusercontent.com/3076261/186295567-ac163c82-b52c-4cd7-9e99-bb6a283ebd3b.png)

**Comment by kinueng** [comment]
The cursor now shows a hand when the cursor hovers over the said elements
